config/locales/en/errors.yml

```diff
@@ -12,6 +12,10 @@ en:
       messages:
         invalid_date: "is not in the correct format"
       models:
+        image_data:
+          attributes:
+            file:
+              blank: cannot be uploaded. Choose a valid JPEG, PNG, SVG or GIF.
```

Comment:
So I think actually what would best here instead would be:

```
en:
  file:
    errors:
      messages:
        blank: "cannot be uploaded. Choose a valid %{extension_list}."
```

and then using 

```
I18n.t("file.errors.blank", extension_list:)
```

for all the uploaders within Whitehall. This could be a different PR though since it would add extra lines that aren't too pertinent to the core changes in this PR.